mongo/with_transactions_test.go

```diff
@@ -186,3 +167,74 @@ func TestConvenientTransactions(t *testing.T) {
 		})
 	})
 }
+
+func setupConvenientTransactions(t *testing.T) *Client {
```

Comment:
Remind me why we're not using the new mongo testing package here?

Reply:
The tests that rely on `WithTransaction` timing out change the timeout to something much smaller than 120 seconds so they don't hang for 2 minutes. This timeout is private though, and it's part of the spec to not make it configurable, so they need to stay in this package.

Reply:
Yuk, ok.